docs/user-documentation/searching.md

```diff
@@ -0,0 +1,62 @@
+# Setup and Configure Search
+
+Islandora 8 comes with the Drupal 8 [Search API](https://www.drupal.org/project/search_api) and [SOLR](https://www.drupal.org/project/search_api_solr) modules enabled with a corresponding SOLR instance. This guide gives an overview to the setup provided by the claw-playbook. Much more detail is available in the [Search API documentation](https://www.drupal.org/docs/8/modules/search-api). Another helpful resource is [Adam Fuch's "Drupal 8 Custom Site Search with Search API"](https://www.electriccitizen.com/blog/drupal-8-custom-site-search-search-api) (https://www.electriccitizen.com, 2018-01-10; last accessed 2019-03-08).
+
+## Indexing Islandora 8 with SOLR
+
+To access the search indexing settings navigate to '/admin/config/search/search-api'.
+
+![Screenshot of the search-api configuration page.](../assets/search-settings-page.png)
+
+### SOLR Server Configuration
+
+Use the 'Solr Server' link to view the SOLR server's configuration and the 'Default Solr content index' link to view or update the index settings. In most cases, where the site was built using ansible, the 'Solr Server' settings that were configured during installation should be left alone.
+
+### SOLR Index Configuration
+
+The Default Solr content index user interface is divided into four tabs: 'View', 'Edit', 'Fields', and 'Processors'.
+
+#### View Tab
+
+The 'View' tab gives an overview of the index and its status:
+- 'Index status' is a progress bar that indicates how many items have been indexed.
+- 'Status' allows you to enable/disable the index from being used.
+- 'Datasource' indicates what types of entities are being indexed and how many have.
+- 'Tracker' states which item tracker is being used.
+- 'Server' provides a link to the SOLR server's configuration page.
+- 'Server index status' is a count of the total number of items returned from an unfiltered empty search. Generally speaking, this should match the total number of items indexed. See the [Drupal 7 Search API FAQ](https://www.drupal.org/node/2009804#server-index-status) for more details.
```

Comment:
Should the link say Drupal 8 search api FAQ? Or is that the old documentation?

Reply:
Yes, that link was for the Drupal 7 Search API (which is were I found out about it); but I see that this FAQ was pulled over into the Drupal 8 documentation, so I will update the link.